### PR TITLE
dcache-core: refactoring slf4j logging messages

### PR DESCRIPTION
**Pull request by marisanest**
Motivation: with normal string concatenations in log-messages strings are always build, regardless if log-level is activated or not. with parameterized log-messages the strings only become build, when the log-level is activated;

Modification: using placeholder with parameterized messages instead of string concatenations

Result: gained efficiency

Signed-off-by: marisanest <marisanest@mailbox.org>

### CODE REVIEW

**Comment by paulmillar** [comment]
ok to test

File: modules/dcache/src/main/java/diskCacheV111/hsmControl/flush/driver/AlternateFlush.java [hunk 1]
```diff
@@ -539,13 +534,12 @@ private int countTotalPendingPool( HsmFlushControlCore.Pool pool ){
          //
          if( list.size() < 2 ){
              if(_ntf) {
-                 _log.info("nextToFlush : currently not enough pools to write on (" + list
-                         .size() + ")");
+                 _log.info("nextToFlush : currently not enough pools to write on ({}", list.size() + ")");
```
Comment (kofemann) [inline modules/dcache/src/main/java/diskCacheV111/hsmControl/flush/driver/AlternateFlush.java]: A-ha!

**Comment by kofemann** [comment]
retest this please

**Comment by kofemann** [comment]
@marisanest Thanks!

**Pull request merged**